CRITICAL - 2024-10-10 14:34:20 --> Error: Class "CI_Controller" not found
[Method: GET, Route: produk]
in APPPATH\Controllers\ProdukController.php on line 3.
 1 SYSTEMPATH\Autoloader\Autoloader.php(317): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\app\\Controllers\\ProdukController.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\ProdukController')
 4 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\ProdukController')
 5 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\ProdukController', true)
 6 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:36:48 --> Error: Class "App\Controllers\Controller" not found
[Method: GET, Route: produk]
in APPPATH\Controllers\Produk.php on line 6.
 1 SYSTEMPATH\Autoloader\Autoloader.php(317): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\app\\Controllers\\Produk.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Produk')
 4 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Produk')
 5 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Produk', true)
 6 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:37:08 --> Error: Cannot call constructor
[Method: GET, Route: produk]
in APPPATH\Controllers\Produk.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Produk->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:37:36 --> ErrorException: Undefined property: App\Controllers\Produk::$load
[Method: GET, Route: produk]
in APPPATH\Controllers\Produk.php on line 12.
 1 APPPATH\Controllers\Produk.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Produk::$load', 'C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\app\\Controllers\\Produk.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Produk->__construct()
 3 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:45:43 --> Error: Class "App\Models\ProductModel" not found
[Method: GET, Route: products]
in APPPATH\Controllers\ProductController.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:46:10 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products/index.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products/index.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products/index', [], true)
 3 APPPATH\Controllers\ProductController.php(13): view('products/index', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
ERROR - 2024-10-10 14:54:32 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp8.1.6\htdocs\SAVORIA-TEST\app\Controllers\ProductController.php(32): CodeIgniter\Model->insert(Array)
#8 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\CodeIgniter.php(933): App\Controllers\ProductController->store()
#9 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#10 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp8.1.6\htdocs\SAVORIA-TEST\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp8.1.6\htdocs\SAVORIA-TEST\system\rewrite.php(44): require_once('C:\\xampp8.1.6\\h...')
#15 {main}
CRITICAL - 2024-10-10 14:54:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: products/store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (:name:, :price:, :description:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ProductController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:54:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (\'test\', \'500\', \'test\', \'2024-10-10 14:54:32\', \'2024-10-10 14:54:32\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (\'test\', \'500\', \'test\', \'2024-10-10 14:54:32\', \'2024-10-10 14:54:32\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (:name:, :price:, :description:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ProductController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
CRITICAL - 2024-10-10 14:54:32 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (\'test\', \'500\', \'test\', \'2024-10-10 14:54:32\', \'2024-10-10 14:54:32\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (\'test\', \'500\', \'test\', \'2024-10-10 14:54:32\', \'2024-10-10 14:54:32\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (\'test\', \'500\', \'test\', \'2024-10-10 14:54:32\', \'2024-10-10 14:54:32\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `products` (`name`, `price`, `description`, `created_at`, `updated_at`) VALUES (:name:, :price:, :description:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ProductController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp8.1.6\\htdocs\\SAVORIA-TEST\\public\\index.php')
